File: app/src/main/java/ir/touristland/Models/HotelPrices/ResultItem.java
package ir.touristland.Models.HotelPrices;

import com.google.gson.annotations.SerializedName;

public class ResultItem {

    @SerializedName("totalPrice")
    private TotalPrice totalPrice;

    @SerializedName("onlineReservation")
    private boolean onlineReservation;

    @SerializedName("fullBoard")
    private boolean fullBoard;

    @SerializedName("nameFa")
    private String nameFa;

    @SerializedName("discount")
    private Discount discount;

    @SerializedName("id")
    private int id;

    @SerializedName("beds")
    private String beds;

    @SerializedName("reservationStatus")
    private boolean reservationStatus;

    @SerializedName("priceList")
    private Object priceList;

    public TotalPrice getTotalPrice() {
        return totalPrice;
    }

    public void setTotalPrice(TotalPrice totalPrice) {
        this.totalPrice = totalPrice;
    }

    public boolean isOnlineReservation() {
        return onlineReservation;
    }

    public void setOnlineReservation(boolean onlineReservation) {
        this.onlineReservation = onlineReservation;
    }

    public boolean isFullBoard() {
        return fullBoard;
    }

    public void setFullBoard(boolean fullBoard) {
        this.fullBoard = fullBoard;
    }

    public String getNameFa() {
        return nameFa;
    }

    public void setNameFa(String nameFa) {
        this.nameFa = nameFa;
    }

    public Discount getDiscount() {
        return discount;
    }

    public void setDiscount(Discount discount) {
        this.discount = discount;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getBeds() {
        return beds;
    }

    public void setBeds(String beds) {
        this.beds = beds;
    }

    public boolean isReservationStatus() {
        return reservationStatus;
    }

    public void setReservationStatus(boolean reservationStatus) {
        this.reservationStatus = reservationStatus;
    }

    public Object getPriceList() {
        return priceList;
    }

    public void setPriceList(Object priceList) {
        this.priceList = priceList;
    }

    @Override
    public String toString() {
        return
                "ResultItem{" +
                        "totalPrice = '" + totalPrice + '\'' +
                        ",onlineReservation = '" + onlineReservation + '\'' +
                        ",fullBoard = '" + fullBoard + '\'' +
                        ",nameFa = '" + nameFa + '\'' +
                        ",discount = '" + discount + '\'' +
                        ",id = '" + id + '\'' +
                        ",beds = '" + beds + '\'' +
                        ",reservationStatus = '" + reservationStatus + '\'' +
                        ",priceList = '" + priceList + '\'' +
                        "}";
    }
}

File: app/src/main/java/ir/touristland/Models/HotelPrices/ResultItem.kt
package ir.touristland.Models.HotelPrices

import com.google.gson.annotations.SerializedName

class ResultItem {

    /* @SerializedName("priceList")
    private Object priceList;*/

    @SerializedName("totalPrice")
    var totalPrice: TotalPrice? = null

    @SerializedName("onlineReservation")
    var isOnlineReservation: Boolean = false

    @SerializedName("fullBoard")
    var isFullBoard: Boolean = false

    @SerializedName("nameFa")
    var nameFa: String? = null

    @SerializedName("discount")
    var discount: Discount? = null

    @SerializedName("id")
    var id: Int = 0

    @SerializedName("beds")
    var beds: String? = null

    @SerializedName("reservationStatus")
    var isReservationStatus: Boolean = false

    /*public Object getPriceList() {
        return priceList;
    }

    public void setPriceList(Object priceList) {
        this.priceList = priceList;
    }*/
}